docs/network_firewall.rst

```diff
@@ -177,6 +177,20 @@ Connect to the pfSense WebGUI
 
 .. _intentionally disables LAN access: https://labs.riseup.net/code/issues/7976
 
+Disable Refer Check
+~~~~~~~~~~~~~~~~~~~
+
+Before you can set up the hardware firewall, you will need to disable the
+**Browser HTTP_REFERER enforcement** setting after you login. You will see
+the Setup Wizard but you shoud exit out of it by clicking the pfSense logo
+at the top left corner. Navigate to **System** -> **Advanced** and look for
+the **Browser HTTP_REFERER enforcement** enforcement setting. You should check
+it and then save your settings. After doing so, you should be able to go
+back to **System** and select **Setup Wizard**.
```

Comment:
Hmm, why does this need to be disabled? Did you attempt to use the Alternate hostnames field such that you don't need to disable this enforcement?